lib/iris/_representation/cube_summary.py

```diff
@@ -259,8 +259,17 @@ def __init__(self, cube, shorten=False, name_padding=35):
         vector_dim_coords = [
             coord for coord in dim_coords if id(coord) not in scalar_coord_ids
         ]
+        if cube.mesh is None:
```

Comment:
I've just done this for now, as it was irritating me + making debugging hard (not to easily see when there is a mesh or not).
**This probably belongs in it's own PR.**